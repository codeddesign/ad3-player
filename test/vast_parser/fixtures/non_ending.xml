<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<VAST version="3.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="vast.xsd">
    <Ad id="preroll-1" sequence="1">
        <InLine>
            <AdSystem version="2.0">DART_DFP</AdSystem>
            <AdTitle id="123"/>
            <Description/>
            <Survey/>
        </InLine>
    </Ad>
</VAST>
