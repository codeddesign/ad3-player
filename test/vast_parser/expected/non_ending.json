{
  "vast": {
    "name": "VAST",
    "value": [
      {
        "ad": {
          "name": "Ad",
          "value": [
            {
              "inline": {
                "name": "InLine",
                "value": [
                  {
                    "adsystem": {
                      "name": "AdSystem",
                      "value": "DART_DFP",
                      "attributes": {
                        "version": "2.0"
                      }
                    }
                  },
                  {
                    "adtitle": {
                      "name": "AdTitle",
                      "value": "",
                      "attributes": {
                        "id": "123"
                      }
                    }
                  },
                  {
                    "description": {
                      "name": "Description",
                      "value": "",
                      "attributes": {}
                    }
                  },
                  {
                    "survey": {
                      "name": "Survey",
                      "value": "",
                      "attributes": {}
                    }
                  }
                ],
                "attributes": {}
              }
            }
          ],
          "attributes": {
            "id": "preroll-1",
            "sequence": "1"
          }
        }
      }
    ],
    "attributes": {
      "version": "3.0",
      "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
      "xsi:nonamespaceschemalocation": "vast.xsd"
    }
  }
}
